ui/tests/acceptance/client-dashboard-test.js

```diff
@@ -341,17 +334,14 @@ module('Acceptance | client counts dashboard tab', function (hooks) {
     assert
       .dom('[data-test-alert-banner="alert"]')
       .hasTextContaining(
-        `Warning Vault was upgraded to 1.10.1 on ${format(
-          UPGRADE_DATE,
-          'MMM d, yyyy'
-        )}. We added monthly breakdowns and mount level attribution starting in 1.10, so keep that in mind when looking at the data. Learn more here.`
+        `Warning Vault was upgraded to 1.10.1 on Aug 1, 2022. We added monthly breakdowns and mount level attribution starting in 1.10, so keep that in mind when looking at the data. Learn more here.`
       );
   });
 
   test('Shows empty if license start date is current month', async function (assert) {
     // TODO cmb update to reflect new behavior
```

Comment:
@hellobontempo is this TODO still relevant? The tests seem to pass but I'm not sure if this meant to check more things?

Reply:
Oish yeah. I noticed this on my last passthrough and wondered the same thing 😅 

Among the miscellany of other bugs, I don't think I ever got an answer from the backend on what the expected response in this case (or perhaps forgot to ask). Previously data wasn't available for a customer until the first completed month, but that should no longer be the case.

This test can be deleted! But a good reminder of something that should be taken into account for the latest backend work on activity log testing